modules/ejbca-ejb-cli/src/org/ejbca/ui/cli/ca/CaImportCertDirCommand.java

```diff
@@ -174,7 +179,21 @@ public CommandResult execute(final ParameterContainer parameters) {
 
             // Fetch CA info
             final CAInfo caInfo = getCAInfo(getAuthenticationToken(), caName);
-            final X509Certificate cacert = (X509Certificate) caInfo.getCertificateChain().iterator().next();
+            
+            // Override the CA certificate if provided in the options
+            X509Certificate cacert = (X509Certificate) caInfo.getCertificateChain().iterator().next();
+
+            if ( caCertFile != null) {
+                final File fileCaCertFile = new File(caCertFile);
+                if (!fileCaCertFile.isFile()) {
+                    log.error("CA Certificate file '" + caCertFile + "' is not found. Will use current certificate for the CA.");
```

Comment:
This should return a failure here:
return CommandResult.CLI_FAILURE;
since the arguments were not supplied properly, let the used know and fix the user error.

Reply:
Yep, good point. Will fix.